context/llpcCompiler.h

```diff
@@ -263,6 +263,42 @@ class Compiler: public ICompiler
     LLPC_DISALLOW_DEFAULT_CTOR(Compiler);
     LLPC_DISALLOW_COPY_AND_ASSIGN(Compiler);
 
+    // Class of object to manage checking and updating shader caches for graphics pipeline.
+    class GraphicsShaderCacheChecker
```

Comment:
I agree we need a separate class for per shader stage cache checker.
because we will extend it to support compile pipeline in shader module. David Zhou is working on it now.
but I prefer not to use nested class.

Reply:
I have un-nested the class.